db/write_callback_test.cc

```diff
@@ -84,7 +84,32 @@ class MockWriteCallback : public WriteCallback {
   bool AllowWriteBatching() override { return allow_batching_; }
 };
 
-TEST_F(WriteCallbackTest, WriteWithCallbackTest) {
+class WriteCallbackPTest
+    : public WriteCallbackTest,
+      public ::testing::WithParamInterface<
+          std::tuple<bool, bool, bool, bool, bool, bool, bool>> {
+ public:
+  void SetUp() override {
```

Comment:
You could do the same thing in a default ctor; there's no need to use `SetUp` in this case.